gulpfile.js

```diff
@@ -33,37 +33,48 @@ gulp.task('liveResetting', function() {
 });
 
 
-gulp.task('moveDependencies', function() {
-   return gulp.src('node_modules/**/*')
+gulp.task('moveNodeDependencies', function() {
+  return gulp.src('node_modules/**/*')
```

Comment:
@mok4ry @Lettuceman44 seeing that spacing diff made me think of something - we should really get some style guides going. I suggest the following:

Javascript: https://google.github.io/styleguide/javascriptguide.xml
HTML/CSS: https://google.github.io/styleguide/htmlcssguide.xml

I've used google's Java style guide in the past and it worked for me, so how do folks feel about that. just so everything is standardized and therefore understandable to the rest of the community.